AssertionError: unkown user

https://sentry.4teamwork.ch/sentry/onegov-gever/issues/1191/

```
AssertionError: unkown user
(21 additional frame(s) were not displayed)
...
  File "home/zope/eggs/opengever.core-4.10.4-py2.7.egg/opengever/bumblebee/service.py", line 34, in get_representation_url
    obj, format_name, filename)
  File "home/zope/eggs/ftw.bumblebee-3.0.1-py2.7.egg/ftw/bumblebee/service.py", line 187, in get_representation_url
    checksum, format_name, filename)
  File "home/zope/eggs/ftw.bumblebee-3.0.1-py2.7.egg/ftw/bumblebee/service.py", line 197, in _get_representation_url_by_checksum
    'access_token': get_access_token_by_checksum(checksum)}
  File "home/zope/eggs/ftw.bumblebee-3.0.1-py2.7.egg/ftw/bumblebee/hashing.py", line 46, in get_access_token_by_checksum
    content=checksum + get_user_salt())
  File "home/zope/eggs/ftw.bumblebee-3.0.1-py2.7.egg/ftw/bumblebee/usersalt.py", line 16, in get_user_salt
    assert userid, 'unkown user'
```

--- a/docs/HISTORY.txt
+++ b/docs/HISTORY.txt
@@ -5,6 +5,9 @@ Changelog
 4.11.0 (unreleased)
 -------------------
 
+- Fixed permission declaration for the documents tooltip view.
+  [phgross]
+
 - Fixed showroom identifier of preview thumbnails in search template.
   [phgross]
 

--- a/opengever/document/widgets/configure.zcml
+++ b/opengever/document/widgets/configure.zcml
@@ -20,7 +20,7 @@
     <browser:page
         name="tooltip"
         for="opengever.document.behaviors.IBaseDocument"
-        permission="zope.Public"
+        permission="zope2.View"
         class=".tooltip.TooltipView"
         template="tooltip.pt"
         />

